package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "license": "Apache License 2.0",
   "config": {
-    "webservice_version": "1.7.0-beta.1"
+    "webservice_version": "1.7.0-beta.2"
```

Comment:
RIP

Reply:
I'm guessing this is why my tests are failing. :(